packages/deployments/contracts/contracts/core/connext/facets/RoutersFacet.sol

```diff
@@ -605,6 +605,13 @@ contract RoutersFacet is BaseConnextFacet {
     // Sanity check: amount can be deducted for the router.
     if (routerBalance < _amount) revert RoutersFacet__removeRouterLiquidity_insufficientFunds();
 
+    // If it is the canonical domain, decrease custodied value
+    if (s.domain == canonical.domain && s.caps[key] > 0) {
+      // NOTE: safe to use the amount here because routers should always supply liquidity
+      // in canonical asset on the canonical domain
+      s.custodied[_local] -= _amount;
```

Comment:
If cap is 0 when some funds were deposited, and later cap is increased. Routers and other entities may not be able to withdraw because `s.custodied[_local] -= _amount;` can revert.

Reply:
instead i chose to initialize custodied, which means this should always be accurate while its tracked

Reply:
yeah, a better solution.